### ISSUE TITLE
Fullscreen Preview

### ISSUE THREAD

**Issue by D-Maxwell** [TheJoeFin/Simple-QR-Code-Maker#27]
Attempting to scan a QR code from within the app itself has revealed quite troublesome, as, regardless of the colour contrast, the brightness of either part of the image bleeds into the other, making for a blurry mess, which, juxtaposed with a camera's potential lack of clarity, prevents any scanning at all.

This is easily fixed by scaling the image to fill a greater area, as the bleeding then becomes much less relevant.

Having in-app controls for the preview to occupy a greater region of the app is my suggestion. As there is presently a great sum of empty space surrounding the already existing controls, this would not interfere with much else.

**Comment by TheJoeFin** [comment]
hello @D-Maxwell the latest version of Simple QR Code Maker has a revamped code scanning capability. I believe this issue is closed, if there are more things you'd like, feel free to open more issues
